### PR TITLE
Display futures position indicators on chart

### PR DESCRIPTION
**Pull request by el0x1**
## Description
- When no position open it will display potential position of entry and liquidation with dashed line as the user adjusts the amount
- When a user has a position open it will display the entry and liquidation points for that position
- Modifying a position when there is already a position open will update the indicators to show new liquidation and the potential average entry

## Related issue
https://github.com/Kwenta/kwenta/issues/693

### CODE REVIEW

**Review by Tburm — APPROVED** [review]
These changes look great, cleans up the chart quite a bit and leaves room for us to continue improving it.

I tested a bunch of potential positions and open positions and couldn't break anything.

File: sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx [hunk 1]
```diff
@@ -51,7 +51,9 @@ const CombinedPriceChartCard: FC<CombinedPriceChartCardProps> = ({
 				{baseCurrencyKey ? (
 					<ChartData disabledInteraction={disabledInteraction}>
 						<Container {...rest}>
-							<TVChart baseCurrencyKey={baseCurrencyKey} quoteCurrencyKey={quoteCurrencyKey} />
+							{quoteCurrencyKey && (
+								<TVChart baseCurrencyKey={baseCurrencyKey} quoteCurrencyKey={quoteCurrencyKey} />
+							)}
```
[the remark this answers is not in the document]
Comment (el0x1) [reply sections/exchange/TradeCard/Charts/CombinedPriceChartCard.tsx:56]: I think this component was previously used for Synth trades and could be any pairing, ETH / LINK for example, both params were necessary. I see you've removed this component in your 782 PR so this change will be redundant anyway.

**Comment by platschi** [comment]
@el0x1 Thank you!  Could you resolve the conflicts?

Once done it gets my stamp of approval 👍

**Comment by el0x1** [comment]
@platschi conflicts now resolved

**Review by platschi — APPROVED** [review]

**Pull request merged**